docs/guide/configuration-options.md

```diff
@@ -17,7 +17,6 @@ static method called to initiate a new instance of HyperFormula.
 // define options 
 const options = {
     licenseKey: 'gpl-v3',
-    precisionRounding: 10,
```

Comment:
I wouldn't remove it here - it's a reasonable example of config option, but I have no strong opinion on that.

Reply:
I agree, putting it back: 8ee4f1c